nixos/modules/services/continuous-integration/gitlab-runner.nix

```diff
@@ -0,0 +1,51 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.gitlab-runner;
+  configFile = pkgs.writeText "config.toml" cfg.configText;
+in
+{
+  options.services.gitlab-runner = {
+    enable = mkEnableOption "Gitlab Runner";
+
+    configText = mkOption {
+      description = "Verbatim config.toml to use";
```

Comment:
I was looking for something similar but only found the `toJSON` builtin.
In the end I decided that it might be easier for to user to use the verbatim gitlab runner config.

As far as I understand you can still use nix variables inside the verbatim text?